tensorflow_datasets/image/lsun.py

```diff
@@ -96,9 +97,10 @@ class Lsun(tfds.core.GeneratorBasedBuilder):
       tfds.core.BuilderConfig(  # pylint: disable=g-complex-comprehension
           name=category,
           description="Images of category %s" % category,
-          version=tfds.core.Version("3.0.0"),
+          version=tfds.core.Version("3.0.1"),
```

Comment:
Since, the dataset now has a completely new `BuilderConfig`, I think it would be better to version it `3.1.0`

Reply:
Sure, done in 6686f37